pkg/driver/helmdriver.go

```diff
@@ -70,12 +73,25 @@ func (d *helmDriver) Install(ctx context.Context,
 		return fmt.Errorf("loading helm chart %s: %w", name, err)
 	}
 
+	// Update values with imagePullSecrets
+	secretvals, err := d.secretAuth.GetSecretValues(ctx)
+	if err == nil {
```

Comment:
What if err != nil?

Reply:
So in cases where their are enviroments that don't need imagePullSecrets, such as registry mirroring to a local registry/or packages in a public registry we want to helm install to continue but not populate the imagePullSecrets value in the chart.

Reply:
Ok